alignak/comment.py

```diff
@@ -55,19 +55,31 @@ class Comment(AlignakObject):
     It contains data like author, type, expire_time, persistent etc..
     """
 
-    properties = {
-        'entry_time':   IntegerProp(),
-        'persistent':   BoolProp(),
-        'author':       StringProp(default='(Alignak)'),
-        'comment':      StringProp(default='Automatic Comment'),
-        'comment_type': IntegerProp(),
-        'entry_type':   IntegerProp(),
-        'source':       IntegerProp(),
-        'expires':      BoolProp(),
-        'expire_time':  IntegerProp(),
-        'can_be_deleted': BoolProp(default=False),
-        'ref':  StringProp(default='')
-    }
+    properties = AlignakObject.properties.copy()
+    properties.update({
+        'entry_time':
+            IntegerProp(default=0),
```

Comment:
I clearly understand the pros and cons of default value and I set some where I thought it is necessary